pygamelib/board_items.py

```diff
@@ -1066,6 +1066,7 @@ def can_move(self):
         """
         return False
 
+    @property
     def inventory_space(self):
         """Return the size of the Immovable Item for the
```

Comment:
What do you think of:
```
Return the size that the Immovable item takes in the :class:`~pygamelib.engine.Inventory`.
```
?

Reply:
> What do you think of:
> 
> ```
> Return the size that the Immovable item takes in the :class:`~pygamelib.engine.Inventory`.
> ```
> 
> ?

That works! I will work on all of these changes in a few hours, if that is okay.